### PR TITLE
feat(examples): add react-relay example

### PR DESCRIPTION
**Pull request by notrab**
# Description

## Todos
- [ ] Add README
- [ ] Add message list component + fragment

# Type of change

- [x] 🚀 Feature

### CODE REVIEW

File: examples/react-relay/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+type Mutation {
+  """Create a Message"""
+  messageCreate(input: MessageCreateInput!): MessageCreatePayload
+
+  """Delete a Message by ID or unique field"""
+  messageDelete(by: MessageByInput!): MessageDeletePayload
+
+  """Update a Message"""
```
Comment (hhthacker) [inline examples/react-relay/schema.graphql:84]: curious if these comments are all generated from somewhere? while some are useful content, a lot of the ones on the fields seem to add little/no value, like `Create a Message` for `messageCreate` :D :D

File: examples/react-relay/src/__generated__/IndexPageContainerMessagesQuery.graphql.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
+/**
+ * @generated SignedSource<<e3c801c4201d1daae77623fc34fbaae4>>
+ * @lightSyntaxTransform
+ * @nogrep
+ */
+
```
Comment (hhthacker) [inline examples/react-relay/src/__generated__/IndexPageContainerMessagesQuery.graphql.js:6]: these generated files are specific to react-relay?

File: examples/react-relay/schema.graphql (hunk 1 above)
Comment (notrab) [reply examples/react-relay/schema.graphql:84]: These comments are generated by Grafbase. There's an issue in Linear to make sure they're helpful. Maybe something we can look at this month?

File: examples/react-relay/src/__generated__/IndexPageContainerMessagesQuery.graphql.js (hunk 2 above)
Comment (notrab) [reply examples/react-relay/src/__generated__/IndexPageContainerMessagesQuery.graphql.js:6]: Yeah these are generated by Relay. I will provide a `script` and add a note to README about this. Good spot.

**Pull request closed without merging**